tensorrt_llm/_torch/pyexecutor/model_engine.py

```diff
@@ -288,6 +288,37 @@ def get_rank_model_storage(model):
     return total_bytes
 
 
+def _filter_cuda_graph_batch_sizes(cuda_graph_batch_sizes: list[int],
```

Comment:
A simple unit test would be nice.

Reply:
I think it's already implicitly covered by a bunch of tests that enable padding. We can test the function in a standalone way though.